packages-ts/gauntlet-terra-contracts/src/commands/abstract/executionWrapper.ts

```diff
@@ -13,46 +21,68 @@ export interface AbstractInstruction<Input, ContractInput> {
   makeInput: (flags: any, args: string[]) => Promise<Input>
   validateInput: (input: Input) => boolean
   makeContractInput: (input: Input) => Promise<ContractInput>
+  beforeExecute?: (context: BeforeExecutionContext<Input, ContractInput>) => () => Promise<void>
   afterExecute?: (response: Result<TransactionResponse>) => any
 }
 
-export const instructionToCommand = (instruction: AbstractInstruction<any, any>) => {
+export const instructionToCommand = <Input, ContractInput>(instruction: AbstractInstruction<Input, ContractInput>) => {
```

Comment:
Maybe stick to the naming we have in the commands files, i.e. `CommandInput` and `ContractInput`?